### PR TITLE
Refactor tests

### PR DESCRIPTION
**Pull request by gerardogzze**
Test

### CODE REVIEW

**Comment by luisvalenzo** [comment]
Please add a description about the purpose of the PR

File: cypress/e2e/testWithPOM.js [hunk 1]
```diff
@@ -44,7 +44,9 @@ describe('Test with Page Object Model', () => {
     })
 
     navigateTo.datepickerPage()
-    onDatePickerPage.selectCommonDatepickerDateFromToday(4)
+    onDatePickerPage.selectCommonDatepickerDateFromToday(10).then(dateAssert => {
+      expect(dateAssert).to.be.true
+    })
     onDatePickerPage.selectDatepickerWithRangeFromToday(50, 51)
```
Comment (luisvalenzo) [inline cypress/e2e/testWithPOM.js:50]: Maybe you can delete this code as the assertion was made previously

File: cypress/e2e/testWithPOM.js [hunk 2]
```diff
@@ -67,7 +69,13 @@ describe('Test with Page Object Model', () => {
 
   it('Create a new record with first name and last name', () => {
```
Comment (luisvalenzo) [inline cypress/e2e/testWithPOM.js:70]: Is there a way that you can assert that the name was submitted successfully?

File: cypress/e2e/testWithPOM.js (hunk 1 above)
Comment (luisvalenzo) [reply cypress/e2e/testWithPOM.js:50]: I see this is for a range, maybe u can do other test for that purpose

**Comment by gerardogzze** [comment]
Que onda, compa!

I appreciate your time in reviewing my pull request. This PR aims to enhance the functionality of the test suite by making the following changes:

- Moving the assertion from the 'datepickerPage.js' to the 'testWithPOM.js' within 'Submit Inline and Basic form and select tomorrow from the calendar' 

- Updating the parameter value in 'selectCommonDatepickerDateFromToday()' on the 'datepickerPage.js' 

- Implementing a data-driven approach in the 'Create a new record with first name and last name' test 

Thank you for your valuable time and consideration!

Best regards,
Gerry

**Review by luisvalenzo — APPROVED** [review]

**Pull request merged**